src/components/Navbar/__test__/TopNavbar.test.tsx

```diff
@@ -49,9 +61,20 @@ describe("Navbar", () => {
     expect(screen.getByText(/View All/i)).toBeInTheDocument();
   });
 
-  it("should render profile in the Navbar", () => {
+  it("should render profile icon if loggedin r", () => {
     render(<MockNavbar />);
-    const ProfileElement = screen.getByText(/Profile/i);
+    const isloggedin = localStorage.getItem("isloggedin");
+    const ProfileElement = screen.getByTestId("profileImage");
+    if (isloggedin === "true") {
+      // eslint-disable-next-line jest/no-conditional-expect
+      expect(ProfileElement).toBeInTheDocument();
+      // eslint-disable-next-line jest/no-conditional-expect
+      expect(ProfileElement).toHaveAttribute("src", "man.png");
+    } else {
+      // eslint-disable-next-line jest/no-conditional-expect
+      expect(ProfileElement).toHaveAttribute("src", "profile.svg");
+    }
+
```

Comment:
we also need a counter test case which has isLoggedIn false

Reply:
added test for not loggedin case